src/types/index.ts

```diff
@@ -1,3 +1,4 @@
+import { CookieOptions } from 'nuxt3/dist/app/composables/cookie'
```

Comment:
@danielroe To confirm with you, is this the right way to get this type?

Reply:
I just tried using the type without any import, this results in the build error 

`error TS4033: Property 'cookie' of exported interface has or is using private name 'CookieOptions'.`

Reply:
With `nuxt3` import it resulted into a build error, I've updated it with the `#app` alias but we still have a warning on build:

```
WARN  Inlining implicit external #app/composables/cookie                                                                                                                    

'#app/composables/cookie' is imported by src/types/index.ts, but could not be resolved – treating it as an external dependency
```

I'm gonna wait for Daniel validation on this.

Reply:
I put #app in the build.config.ts, which seems to resolve the problem (I am not familiar with rollup)

`externals: ['@nuxt/kit', '@nuxt/schema', 'vue', '#app']`

Reply:
Thanks a lot @danielroe, it works perfectly!